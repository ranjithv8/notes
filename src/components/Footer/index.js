import React from 'react';

const Footer = () => (
  <footer className="footer">
    <div className="content has-text-centered">
      <p>
        <strong>Notes</strong> by <a href="https://jgthms.com">Sunil Hari</a>
        <a href="http://opensource.org/licenses/mit-license.php">MIT</a>
      </p>
    </div>
  </footer>
)
export default Footer;