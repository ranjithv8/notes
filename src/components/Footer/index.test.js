import React from 'react';
import Footer from './index';
import renderer from 'react-test-renderer';

test('It Renders Footer without error', () => {
  const component = renderer.create(<Footer />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});