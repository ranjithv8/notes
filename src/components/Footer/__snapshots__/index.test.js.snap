// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It Renders Footer without error 1`] = `
<footer
  className="footer"
>
  <div
    className="content has-text-centered"
  >
    <p>
      <strong>
        Notes
      </strong>
       by 
      <a
        href="https://jgthms.com"
      >
        Sunil Hari
      </a>
      <a
        href="http://opensource.org/licenses/mit-license.php"
      >
        MIT
      </a>
    </p>
  </div>
</footer>
`;
